spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -47,6 +47,9 @@ An alternative may be given a priority for the application:
 * by placing the `@Priority` annotation on the bean class of a managed bean, or
 * by placing the `@Priority` annotation on the bean class that declares the producer method, field or resource.
 
+For a custom implementation of the `Bean` interface defined in <<bean>>, the container calls `isAlternative()` to determine whether the bean is an alternative and the bean must implement the <<prioritized, `Prioritized` interface>> to select this alternative for the entire application and give it a priority.
```

Comment:
I see several problems here. Firstly you probably meant to say that only if it is alternative than it should implement prioritized interface. Secondly `must implement` is IMHo wrong. What would mean any existing alternative custom bean would become invalid unless it implements prioritized. How about this?:
_Custom bean implementations which are also alternatives may implement <<prioritized, `Prioritized` interface>> in which case they will be enabled for entire application with given priority._